lib/transport.rb

```diff
@@ -0,0 +1,24 @@
+class Transport
+  include Comparable
+
+  attr_accessor :max_weight, :speed, :available
```

Comment:
:max_weight and :speed comes from constants and should not be editable, please use attr_reader here.